purchase_exception/data/purchase_exception_data.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <!-- Test Purchase Exceptions Scheduler-->
+    <record model="ir.cron" forcecreate="True"
+            id="ir_cron_test_po_order_except">
+        <field name="name">Purchase: Test Draft Orders Exception</field>
+        <field name="model_id" ref="purchase.model_purchase_order"/>
+        <field name="state">code</field>
+        <field name="code">model.test_all_draft_orders()</field>
+        <field name="user_id" ref="base.user_root"/>
+        <field name="interval_number">20</field>
+        <field name="interval_type">minutes</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="active" eval="False" />
+    </record>
+
+    <record id ="po_excep_no_email" model="exception.rule">
+        <field name="name">No email on vendor</field>
+        <field name="description">No Email for Vendor</field>
+        <field name="sequence">50</field>
+        <field name="model">purchase.order</field>
+        <field name="code">if not purchase.partner_id.email:
```

Comment:
Please, use "self" instead of purchase, as purchase is derecated and should be removed in next version.

Reply:
Done.
Thanks for the the feedback.